docs/src/components/App.js

```diff
@@ -65,10 +65,6 @@ export default function App(props: Props): Node {
               <Box padding={4} mdPadding={6} lgPadding={8} width="100%">
                 {children}
               </Box>
-            </Box>
-          </Box>
-          {document.location.href.includes('netlify') ? (
```

Comment:
Lesson learned - our local docs should looks as similar as possible as the ones hosted on netlify. So we discover these issues early on.